library/core/src/iter/adapters/step_by.rs

```diff
@@ -1,7 +1,7 @@
 use crate::{
     intrinsics,
     iter::{from_fn, TrustedLen, TrustedRandomAccess},
-    num::NonZeroUsize,
```

Comment:
can you avoid these changes in the stabilization PR and *only* make it stable (and remove `feature` annotations)? this can be done in the future just fine and will make it easier to reviert